src/entries/popup/components/ChainBadge/ChainBadge.tsx

```diff
@@ -138,31 +144,31 @@ const ChainBadge = ({
         ...(customChainIdsToAssetNames[chainId] ? {} : { boxShadow }),
       }}
     >
-      {customChainIdsToAssetNames[chainId] ? (
```

Comment:
previously we were preferring custom badge assets over local, was this intentional?

Reply:
@walmat I think this was so that we could resolve the chain badge from the `rainbow-me/assets` repo for custom networks, but it looks like we've appended / forgot to remove supported networks from `customChainIdsToAssetNames` and didn't anticipate this side effect. I suppose that mapping should only include unsupported chains so that we rely on the local badge